php/class-wp-cli.php

```diff
@@ -287,6 +287,25 @@ public static function confirm( $question, $assoc_args = array() ) {
 	}
 
 	/**
+	 * Prompt before continuing.
+	 *
+	 * @param string $question The question to pose
+	 * @param bool $default The default response
+	 * @param array $answers when the users responds with $answers[0] return false, with $answers[1] return true
+	 * @param bool $hide Hide the user's response
+	 * @return bool
+	 */
+	public static function prompt( $question, $default = false, $answers = array( 'no', 'yes' ), $hide = false ) {
+		$marker = '[' . implode( '/', $answers ) . ']: ';
+
+		do {
+			$answer = cli\prompt( $question, $default, $marker, $hide );
+		} while ( ! in_array( $answer, $answers ) );
+
+		return $answers[ 1 ] === $answer;
+	}
+
+	/**
```

Comment:
Er, I don't think we need this abstraction. Can you just include in your method?